torchft/process_group_test.py

```diff
@@ -391,18 +394,19 @@ def test_error_swallowing_process_group_wrapper(self) -> None:
         wrapper = ErrorSwallowingProcessGroupWrapper(pg)
         self.assertIs(wrapper.parent, pg)
 
-        works = _test_pg(wrapper)
+        works = run_collective(pg=wrapper, collective="allreduce")
```

Comment:
this seems like a pretty big decrease in coverage?

Reply:
Added functionality back